### ISSUE TITLE
turret.local not reachable

### ISSUE THREAD

**Issue by mWollenhaupt** [joranderaaff/portalturret#2]
Hi Joran,

classic mistake - Implemented the whole project and now fail with your software (v2). Could it be that there are still entries to be made in the code here or should everything run out of the box?
I just can't connect to http://turret.local, not even via direct IP. Connecting via SSID and password work, but I just can't reach the device?
What versions of your dependencies (libraries) are you using? Have there been any serious changes?

**Comment by joranderaaff** [comment]
Did you upload the data folder with LittleFS? That package contains all HTML files. If it's missing, the webpage will give an error like that.

If that fails, I'd connect directly and try the IP address instead of te device instead of the .local shorthand

**Comment by mWollenhaupt** [comment]
Hi, thanks for the quick reply

My mistake: I did not know the difference between SPIFFS and LittleFS. The data can be processed now.

What is still problematic:
There is no settings.txt by default - I think the idea is that the file is generated automatically once a first configuration has taken place? Without wifi credentials, I can connect to the portal turret - wifi, but can't get to turret.local or into /setup.

I got it to work by creating the settings.txt (\<SSID\>\r\n\<password\>) by hand and loading it directly into the 8266's memory.

Great project
